### DISCUSSION TITLE
Any reason not to use an existing tool?

### DISCUSSION THREAD

**Discussion by OneOfMany07** [memtest86plus/windows-installer#4] [General]
Any reason not to use an existing tool?  I've seen a number of ISO to USB tools, like "Rufus".  Last I saw many Linux distros were switching their setup media creation process to use them.

Here are a few of the top items from https://alternativeto.net/software/rufus/  and what seems special about it.

Name I remembered after told to use once - https://rufus.ie/en/
Again, told to use it once - https://unetbootin.github.io/
SDcards or USB, very simple UI - https://www.balena.io/etcher/
Can create multiboot USB with many tools on it, with boot menu - https://www.pendrivelinux.com/yumi-multiboot-usb-creator/

I figure the less code you have to monitor and fix, the better.  I'd enjoy the flexibility allowing multi-boot has, but I have no idea if it limits you or is harder to set up.

**Comment by x86fr** [comment]
The Memtest86+ Windows Installer is based on two existing open-source tools called Syslinux 6.03 (https://wiki.syslinux.org/) and NSIS Installer (https://nsis.sourceforge.io/). There is no specific code to monitor except the small NSIS script. 

We want the Windows USB Installer extremely easy to use and as lightweight as possible (the actual size is < 500 KB).

Rufus is the cleanest option and we'll try to deal with them after the v6 release. Maybe we can get added like FreeDOS as standard media creation...
Unetbootin and PendriveLinux can consider us for addition at any time. They'll probably do it as soon as v6 final is released.
Balena etcher is a 138MB download just to run a few cmd line scripts (thanks to Electron).
